worker-blacklist

```diff
@@ -0,0 +1,26 @@
+# This file serves as a blacklist for SyTest tests that we expect will fail in
```

Comment:
could these go somewhere other than the root of the repository? There's enough clutter there already.

Reply:
done